src/main/charts/confluence/values.yaml

```diff
@@ -608,6 +608,31 @@ confluence:
     #
     usePodNameAsClusterNodeName: true
 
+  # Use AWS S3 to store attachments. From Confluence 8.1 onwards
+  #
+  s3AttachmentsStorage:
+    # Bucket name to store attachments. If a bucket name is defined, Confluence will automatically
+    # use AWS S3 to store attachments. Only bucket name is required, not the full arn. This can also be defined in
```

Comment:
I am not sure we need to include

```This can also be defined in
    # `confluence.additionalEnvironmentVariables` as `CONFLUENCE_FILESTORE_ATTACHMENTS_S3_BUCKET_NAME`
    # or `confluence.additionalJvmArgs` as `-Dconfluence.filestore.attachments.s3.bucket.name=<bucket-name>
```

this here. Maybe in docs? But if they are setting the value here, why customer need to understand the internals?